<ui version="4.0" >
 <class>OptGeneralDlg</class>
 <widget class="QDialog" name="OptGeneralDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>General</string>
  </property>
  <widget class="QWidget" name="Layout3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>306</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="TextLabel12" >
        <property name="text" >
         <string>Language:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_language" />
      </item>
     </layout>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="TextLabel1" >
      <property name="text" >
       <string>Language settings go into effect the next time you start Qfsm.</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
